cmd/client/txgen/README.md

```diff
@@ -1 +1,2 @@
-The txgen program is used to simulate transactions and hit the Harmony network to loadtest its performance and robustness.
\ No newline at end of file
+The txgen program is used to simulate transactions and hit the Harmony network to loadtest its performance and robustness.
+You can send txns to specific shards 1,2,3 or to shard 0. Sending it to shard 0, broadcasts txns to all the shards.
```

Comment:
broadcasts txns to all shards is a unwanted effect we should fix. Please add a note or todo.

Reply:
https://github.com/harmony-one/harmony/issues/847

Reply:
Done.